### ISSUE TITLE
CodePen snippets not displaying

### ISSUE THREAD

**Issue by baldwindavid** [mvcss/library#7]
Not a big deal, but the component demos within the comments do not work in CodePen. I emailed CodePen and they said they do not support the :sass filter in the HAML. Perhaps they supported it in the past.

**Comment by drewbarontini** [comment]
Yeah, it seems to go off and on with their support of the filter. Thanks for reporting this! We'll fix them up so they work.
